src/sentry/runner/initializer.py

```diff
@@ -366,6 +367,9 @@ def initialize_app(config, skip_service_validation=False):
 
     setup_services(validate=not skip_service_validation)
 
+    if settings.DEMO_MODE:
+        setup_demo_mode_task()
+
```

Comment:
This feels a bit weird to me. As much as possible, we should keep the demo code out of core sentry.

We should probably have a `demo.py` config file that builds off of `server.py`, and just adds the extra task to celerybeat there.

Reply:
@wedamija This might be a silly question, but once I move this to `demo.py`, how do I get it to run that file other than by importing it somewhere here?

Reply:
You can specify different settings files via `--settings` or `DJANGO_SETTINGS_MODULE` env var.